test/functional/wallet_import_rescan.py

```diff
@@ -39,23 +38,17 @@
 class Variant(collections.namedtuple("Variant", "call data rescan prune")):
     """Helper for importing one key and verifying scanned transactions."""
 
-    def try_rpc(self, func, *args, **kwargs):
-        if self.expect_disabled:
-            assert_raises_rpc_error(-4, "Rescan is disabled in pruned mode", func, *args, **kwargs)
```

Comment:
Now there is no longer any test coverage for the "rescan is disabled" cases, but I'm not sure if there's an easy way to add it back. Maybe a `pruneblockchain` RPC call could be added.

Reply:
If a test was added, it should also (or more importantly) test `importmulti`, so I am not actually decreasing test coverage here.

Reply:
I agree that a test would be nice for that case, but can be done in a follow up pull request.

Reply:
> it should also (or more importantly) test importmulti

In case it helps, there is currently a unit test for importmulti behavior with pruning: https://github.com/bitcoin/bitcoin/blob/2d16fb7a2b6a9e5a2535295d2de03e27c2438d1f/src/wallet/test/wallet_tests.cpp#L158